clients/client-python/gravitino/name_identifier.py

```diff
@@ -26,7 +28,13 @@ class NameIdentifier(DataClassJsonMixin):
     _DOT: ClassVar[str] = "."
 
     _name: str = field(metadata=config(field_name="name"))
-    _namespace: Namespace = field(metadata=config(field_name="namespace"))
+    _namespace: Namespace = field(
+        metadata=config(
+            field_name="namespace",
+            encoder=Namespace.to_json,
```

Comment:
what's the purpose of adding this piece of code?

Reply:
Because when deserializing from json, the namespace itself will be a `list`, not a `Namespace`, so the code here is to add encode and decode logic when `dataclass_json` try to handle the DTO. Otherwise, the `namespace` here will be a plain `list`.

Reply:
I see, thanks for the information!